test/run_e2e_test.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+# Terminal output colors
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+CYAN='\033[0;36m'
+NORM='\033[0m'
+# Defaults
+ARCH="aarch64"  # or "armhf"
+OS="ubuntu"     # or "debian"
+DISTRO="dashing"
+RMW="fastrtps"
+RESULT=1        # Default to failure
+
+cleanup(){
+  # Usage: cleanup CONTAINER_NAME RESULT
+  if [[ "$2" -eq 0 ]]; then
+    printf "%b%s%b\n" "$GREEN" "PASS" "$NORM"
```

Comment:
Those printf are not easily readable. What about defining `log` `error` `warn` and use them?
https://repo.or.cz/svn-wrapper.git/blob/HEAD:/svn-wrapper.sh is a good example.